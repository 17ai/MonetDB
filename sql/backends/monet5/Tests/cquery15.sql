--test alias definitions for continuous queries
create table cqresult15(i integer);

create procedure cq_basic15(val int)
begin
	insert into cqresult15 values (val);
end;

start continuous cq_basic15(2) as one_alias;

start continuous cq_basic15(2) as one_aliasother;

start continuous cq_basic15(3) as one_alias; --error

stop continuous cq_basic15; --error

stop continuous one_alias;

stop continuous one_aliasother;

stop continuous one_aliasother; --error

drop procedure cq_basic15;
drop table cqresult15;
