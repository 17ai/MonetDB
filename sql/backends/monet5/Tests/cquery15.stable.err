stderr of test 'cquery15` in directory 'sql/backends/monet5` itself:


# 17:45:14 >  
# 17:45:14 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=31830" "--set" "mapi_usock=/var/tmp/mtest-13813/.s.monetdb.31830" "--set" "monet_prompt=" "--forcemito" "--dbpath=/home/ferreira/MonetDB-trails/BUILD/var/MonetDB/mTests_sql_backends_monet5"
# 17:45:14 >  

# builtin opt 	gdk_dbpath = /home/ferreira/MonetDB-trails/BUILD/var/monetdb5/dbfarm/demo
# builtin opt 	gdk_debug = 0
# builtin opt 	gdk_vmtrim = no
# builtin opt 	monet_prompt = >
# builtin opt 	monet_daemon = no
# builtin opt 	mapi_port = 50000
# builtin opt 	mapi_open = false
# builtin opt 	mapi_autosense = false
# builtin opt 	sql_optimizer = default_pipe
# builtin opt 	sql_debug = 0
# cmdline opt 	gdk_nr_threads = 0
# cmdline opt 	mapi_open = true
# cmdline opt 	mapi_port = 31830
# cmdline opt 	mapi_usock = /var/tmp/mtest-13813/.s.monetdb.31830
# cmdline opt 	monet_prompt = 
# cmdline opt 	gdk_dbpath = /home/ferreira/MonetDB-trails/BUILD/var/MonetDB/mTests_sql_backends_monet5
# cmdline opt 	gdk_debug = 536870922

# 17:45:14 >  
# 17:45:14 >  "mclient" "-lsql" "-ftest" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-13813" "--port=31830"
# 17:45:14 >  

MAPI  = (monetdb) /var/tmp/mtest-13813/.s.monetdb.31830
QUERY = start continuous cq_basic15(3) as one_alias; --error
ERROR = !The continuous procedure one_alias is already registered.
CODE  = 3F000
MAPI  = (monetdb) /var/tmp/mtest-13813/.s.monetdb.31830
QUERY = stop continuous cq_basic15; --error
ERROR = !The continuous query cq_basic15 has not yet started
CODE  = 42000
MAPI  = (monetdb) /var/tmp/mtest-13813/.s.monetdb.31830
QUERY = stop continuous one_aliasother; --error
ERROR = !The continuous query one_aliasother has not yet started
CODE  = 42000

# 17:45:15 >  
# 17:45:15 >  "Done."
# 17:45:15 >  

