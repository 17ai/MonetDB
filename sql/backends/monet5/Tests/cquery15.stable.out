stdout of test 'cquery15` in directory 'sql/backends/monet5` itself:


# 17:45:14 >  
# 17:45:14 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=31830" "--set" "mapi_usock=/var/tmp/mtest-13813/.s.monetdb.31830" "--set" "monet_prompt=" "--forcemito" "--dbpath=/home/ferreira/MonetDB-trails/BUILD/var/MonetDB/mTests_sql_backends_monet5"
# 17:45:14 >  

# MonetDB 5 server v11.28.0
# This is an unreleased version
# Serving database 'mTests_sql_backends_monet5', using 8 threads
# Compiled for x86_64-pc-linux-gnu/64bit with 128bit integers
# Found 15.498 GiB available main-memory.
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008-2017 MonetDB B.V., all rights reserved
# Visit https://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://gup-128-81.cwi.nl:31830/
# Listening for UNIX domain connection requests on mapi:monetdb:///var/tmp/mtest-13813/.s.monetdb.31830
# MonetDB/GIS module loaded
# MonetDB/SQL module loaded
# MonetDB/Timetrails module loaded

Ready.
# SQL catalog created, loading sql scripts once
# loading sql script: 09_like.sql
# loading sql script: 10_math.sql
# loading sql script: 11_times.sql
# loading sql script: 12_url.sql
# loading sql script: 13_date.sql
# loading sql script: 14_inet.sql
# loading sql script: 15_querylog.sql
# loading sql script: 16_tracelog.sql
# loading sql script: 17_temporal.sql
# loading sql script: 18_index.sql
# loading sql script: 20_vacuum.sql
# loading sql script: 21_dependency_functions.sql
# loading sql script: 22_clients.sql
# loading sql script: 23_skyserver.sql
# loading sql script: 25_debug.sql
# loading sql script: 26_sysmon.sql
# loading sql script: 27_rejects.sql
# loading sql script: 39_analytics.sql
# loading sql script: 39_analytics_hge.sql
# loading sql script: 40_geom.sql
# loading sql script: 40_json.sql
# loading sql script: 40_json_hge.sql
# loading sql script: 41_md5sum.sql
# loading sql script: 45_uuid.sql
# loading sql script: 46_profiler.sql
# loading sql script: 50_cquery.sql
# loading sql script: 51_sys_schema_extension.sql
# loading sql script: 60_wlcr.sql
# loading sql script: 72_fits.sql
# loading sql script: 74_netcdf.sql
# loading sql script: 75_lidar.sql
# loading sql script: 75_shp.sql
# loading sql script: 75_storagemodel.sql
# loading sql script: 80_statistics.sql
# loading sql script: 80_udf.sql
# loading sql script: 80_udf_hge.sql
# loading sql script: 85_bam.sql
# loading sql script: 90_generator.sql
# loading sql script: 90_generator_hge.sql
# loading sql script: 99_system.sql

# 17:45:14 >  
# 17:45:14 >  "mclient" "-lsql" "-ftest" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-13813" "--port=31830"
# 17:45:14 >  

#create table cqresult15(i integer);
#create procedure cq_basic15(val int)
#begin
#	insert into cqresult15 values (val);
#end;
#stop continuous one_alias;
#stop continuous one_aliasother;
#drop procedure cq_basic15;
#drop table cqresult15;

# 17:45:15 >  
# 17:45:15 >  "Done."
# 17:45:15 >  

